1.19_linux/runtime/cgo/hitsumabushi_cpu_linux.c

```diff
@@ -1,8 +1,20 @@
 // SPDX-License-Identifier: Apache-2.0
 // SPDX-FileCopyrightText: 2022 The Hitsumabushi Authors
 
-#include <stdint.h>
+#include <stddef.h> // for size_t
+#include <stdint.h> // for int32_t
+#include <sched.h> // for pid_t
 
 int32_t hitsumabushi_getproccount() {
 	return 1;
 }
+
+int32_t hitsumabushi_sched_getaffinity(pid_t pid, size_t cpusetsize, void *mask) {
```

Comment:
Do we really need to expose this for rewriting?

Reply:
Probably not required, indeed. The Sony consoles have an equivalent for this function so I was happy to use it, but it’s really not very important.